Song: remove dead code & `CryptManager.h` include

sukibaby opened the pull request:
We can get rid of the `CryptManager.h` dependency by removing this function which isn't used anywhere.

sukibaby committed:
Remove unused code and CryptManager.h include from Song.cpp

We can get rid of the CryptManager dependency by removing this function which isn't used anywhere.

teejusb commented on src/Song.cpp:
```diff
@@ -1765,26 +1762,6 @@ RString Song::GetCacheFile(RString sType)
 	return "";
 }
 
-RString Song::GetFileHash()
```
We'd also need to remove it from the header

sukibaby replied:
Oh true!

sukibaby committed:
Remove unused file hash members from Song.h

Deleted the m_sFileHash member and GetFileHash() method from the Song class, as they are no longer used.
